### PR TITLE
Added Simulink model backups

### PR DESCRIPTION
**Pull request by IndefiniteBen**
**Reasons for making this change:**

When opening a Simulink model created in an earlier version of Simulink it backs up the model file in a file named `<filename>.<model extension>.<last saved version>` such as `modelName.slx.r2010` and creates a new file in the current Simulink version. With git we can just roll back to a commit before the file changed which gets the same end result.

**Links to documentation supporting these rule changes:** 

https://nl.mathworks.com/help/simulink/gui/simulink-preferences-model-file-pane.html#bsmubvu-1

### CODE REVIEW

File: Global/Matlab.gitignore [hunk 1]
```diff
@@ -20,4 +20,8 @@ sccprj/
 octave-workspace
 
 # Simulink autosave extension
-*.autosave
+.autosave
```
[the remark this answers is not in the document]
Comment (IndefiniteBen) [reply Global/Matlab.gitignore]: Which change are you referring to? It looks like you're referring to: 
`-*.autosave`
`+*.autosave`
but that's the same (apart from EOL/whatever) I didn't intentionally change the .autosave part.
If you mean the `MyModel.slx.Version` part I directly linked to the 'Save backup when overwriting a file created in an older version of Simulink` heading of the linked article.
If this doesn't answer your question I'm not sure I understand.

[the remark this answers is not in the document]
Comment (shiftkey) [reply Global/Matlab.gitignore]: This line has dropped the `*` at the start of the rule. Is that intentional?

[the remark this answers is not in the document]
Comment (IndefiniteBen) [reply Global/Matlab.gitignore]: Oh I see. No it isn't (I even typed it in that line in my above comment). There should be a *

**Comment by shiftkey** [comment]
Closing this out due to inactivity (and conflicts). @IndefiniteBen if you're interested in seeing this land please resolve the conflicts and feedback and we can revisit the changes.

**Pull request closed without merging**